### DISCUSSION TITLE
K8s deployment with External S3 support

### DISCUSSION THREAD

**Discussion by Bytelegion** [Budibase/budibase#5973] [Help]
Hi Community,

Anyone succeeded to deploy Budibase with external S3 endpoint [rather than Minio] for object storage.

**Comment by melohagan** [comment]
Hi @Bytelegion 

It should work with AWS, but otherwise there is limited support at the moment for external S3 buckets: https://github.com/Budibase/budibase/issues/5587

**Reply by Bytelegion** [reply]
Thanks for prompt reply @melohagan , 

Actually i used this for storing attachments  instead of keeping them on Minio server.


I am using Oracle S3 which is compatible s3 service and it worked almost well.

I am able to put and delete files, just not able to read files from s3 because of some wrong end-point.
